cmd/resource-topology-exporter/main_suite_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
I'm not that fond of ginkgo for unit tests (see: pkg/kubeconf/kubelet_config_file_test.go). We use in general `convey` and/or `ginkgo` only for code which we want to merge in other projects (NFD) which already use this infra.
Or, from the other perspective, what do we gain in using ginkgo in unit tests?

Reply:
1. I used PAO project as a reference and I saw we're using ginkgo for unit tests there as well.
2. Consistency - Use the same framework for both e2e & unit test
3. Nit: I'm more experience with ginkgo :)

Reply:
thing is, different tests serve different purposes. So using the same framework for both is actually a double edged sword. Since we don't really have consistency in this project, I don't really mind, but I'm (strongly) inclined to use the plain `testing` module for unit tests, which is usually more than up to the task, unless there are clear reasons to use something more featureful.